codespell_lib/_codespell.py

```diff
@@ -612,15 +617,15 @@ def main(*args):
         print("ERROR: --write-changes cannot be used together with "
               "--regex")
         parser.print_help()
-        return 1
+        return 2
```

Comment:
https://docs.python.org/2/library/os.html#os.EX_USAGE

Reply:
The following seems like it would be the simplest way to use the `sysexits.h` codes:

```python
return 65 # os.EX_USAGE
```

For some reason Python seems to only make these constants available when the actual `sysexits.h` header exists on the system ("Availability: Unix"), but they're exactly the same on every platform I'm aware of (NetBSD, FreeBSD, OpenBSD, glibc, musl). The original BSD man page explicitly specifies the values.

Reply:
I went with `2` because it's what `codespell --foo` gave me (same as `bash --foo`). `EX_USAGE` says it's [available on UNIX](https://docs.python.org/3/library/os.html#os.EX_USAGE), which I assume means it's not on Windows. So I think `2` might be the best option here

Reply:
Meh, I like `sysexits.h` codes, but I guess as long as the wrapping bug (`256`) is fixed, it doesn't matter all that much.

Reply:
Looks like `EX_USAGE==64` on my macOS machine actually (!). Even `brew --foo` gives `1` so who knows... I'm not sure we should allow it to be system-dependent, given that `bash` and `flake8` don't do it. I'd err on the side of Python-ecosystem consistency here.

Reply:
The point of semantic exit codes is that you define ahead of time what they mean (preferably documented) and then use them accordingly. Most other programs aren't really "consistent" -- they generally just return whatever they feel like, either chosen at random or some basic scheme of their own invention. Looking to programs like that for guidance is just propagating the same lack of direction, but alas, it doesn't really matter if the wrapping bug is at least fixed.